### ISSUE TITLE
[MU4 Issue] Score thumbnails are stretched and look blurry on high-res screen

### ISSUE THREAD

**Issue by cbjeukendrup** [musescore/MuseScore#13041]
**Describe the bug**
In Home > Scores, the score thumbnails are stretched, especially when the paper size is very different than the shape of the thumbnail rectangle. For example, here is one of the VTests. The score looks like this:
<img width="460" alt="Scherm­afbeelding 2022-08-27 om 12 54 52" src="https://user-images.githubusercontent.com/48658420/187027091-9e40aee9-a2e7-4570-a6f0-0fbd1c01bdb7.png">
And the thumbnail looks like this:
<img width="228" alt="Scherm­afbeelding 2022-08-27 om 12 54 01" src="https://user-images.githubusercontent.com/48658420/187027072-dc78b121-c333-41d0-8605-9140cbef6824.png">

Additionally, on high-res screens, the thumbnail image looks blurry, because the resolution of the image is not high enough.

**Expected behavior**
That's to be decided by the designers (@Tantacrul, @jessjwilliamson)

**Possible solution**
For the stretching problem: simply adjust some QML code. 
For the resolution problem: the only solution is to store the images at a higher resolution in the mscz files. This will only fix it for new scores, not for existing ones (until you save them in MU4).

**Additional information**
Thumbnails for imported files have no problems, since they go through a different process.

**Comment by Tantacrul** [comment]
Assigning the design of the desired result to @jessjwilliamson. I think we'll save it for 4.x.

**Comment by Tantacrul** [comment]
Correction. I have it targeted at Beta2 for now. If we get to it, that'll be great although it's definitely not a shipblocker.

Thanks @cbjeukendrup !

**Comment by Jojo-Schmitz** [comment]
Hasn't the thumbnail size been doubled recently, in #28238 ?

OTOH this just came up in https://musescore.org/en/node/388116 and it still looks bad in master, so just doubling the size doesn't seem to help here

**Comment by cbjeukendrup** [comment]
Scores that have not yet been saved in master will still have a small-size thumbnail, so that might explain that some people say it still looks bad on master.

**Comment by Jojo-Schmitz** [comment]
True, but saving it in master doesn't help much, not in this case at least:

<img width="249" height="342" alt="Image" src="https://github.com/user-attachments/assets/2a9c2f64-1026-43e9-90a9-9a6c8bb73342" />
